Formula/tinyscheme.rb

```diff
@@ -20,35 +20,38 @@ def install
     share.install("init.scm")
     bin.install("scheme")
   end
+
+  test do
+    assert_match "Usage: tinyscheme", shell_output("#{bin}/scheme -\?", 1)
+  end
 end
 
 __END__
---- a/makefile  2011-01-16 20:51:17.000000000 +1300
-+++ b/makefile  2012-04-08 22:38:11.000000000 +1200
+--- a/makefile
++++ b/makefile
```

Comment:
> Please use the commit reference rather a direct link to master; doing so is a small safety step in case we accidentally remove a patch prematurely or accidentally :smile:.

Got it :+1: